PomoTodo/PomoTodo/Sources/Views/ToDoView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  ToDoView.swift
+//  PomoTodo
+//
+//  Created by 심근웅 on 2/18/25.
+//
+
+import SwiftUI
+
+struct ToDoView: View {
+    let color: [Color] = [Color.indigoNormal, Color.blue, Color.cyan, Color.teal ]
+    var tags: [Tag]
+    @State var todoArr: [Todo]
+    
+    var today: String = {
+       let formatter = DateFormatter()
+        formatter.dateFormat = "M월 dd일 (E)"
+        formatter.locale = Locale(identifier: "ko_KR")
+        return formatter.string(from: Date())
+    }()
```

Comment:
date Extension으로 빼주세요
```suggestion
extension Date {
    var formattedDate: String {
        let formatter = DateFormatter()
        formatter.dateFormat = "M월 dd일 (E)"
        formatter.locale = Locale(identifier: "ko_KR")
        return formatter.string(from: self)
    }
}
```

Reply:
넵 분리 시켰습니다